packages/@aws-cdk/aws-ecs/lib/ec2/ec2-service.ts

```diff
@@ -254,8 +279,8 @@ export class Ec2Service extends BaseService implements IEc2Service {
  * Validate combinations of networking arguments
  */
 function validateNoNetworkingProps(props: Ec2ServiceProps) {
-  if (props.vpcSubnets !== undefined || props.securityGroup !== undefined || props.assignPublicIp) {
-    throw new Error('vpcSubnets, securityGroup and assignPublicIp can only be used in AwsVpc networking mode');
+  if (props.vpcSubnets !== undefined || props.securityGroups !== undefined || props.assignPublicIp) {
```

Comment:
should this be
```
props.vpcSubnets !== undefined
|| props.securityGroup !== undefined
|| props.securityGroups !== undefined
|| props.assignPublicIp
```
?

Reply:
Thanks for catching this! I updated the expression, and added test cases for this function.

I think ideally this would be a method instead of a function. We wouldn't need to pass in `props` and could remove the `securityGroup` condition.